code/scripting/api/objs/file.cpp

```diff
@@ -6,63 +6,72 @@ extern int cfread_lua_number(double *buf, CFILE *cfile);
 namespace scripting {
 namespace api {
 
+cfile_h::cfile_h(CFILE* cfp) : _cfp(cfp) {}
+cfile_h::~cfile_h()
+{
+	if (_cfp && cf_is_valid(_cfp.get())) {
+		Error(LOCATION, "Lua file handle has been left open!\n\nSorry, I can't say anything more than that...");
```

Comment:
Is it possible to say what file name the handle refers to?  If not, could that functionality be added easily?

Reply:
No, unfortunately that is not possible without doing some rather extensive changes to the CFILE system. I wanted to do that for the CFILE leak detection I added some time ago but that information is not stored anywhere at the moment.